scripts/tests/publish_check/test_gpui_packaged_validation.py

```
@@ -0,0 +1,204 @@
+    (crate_dir / "src").mkdir(parents=True)
+    (crate_dir / "src" / "lib.rs").write_text("// test", encoding="utf-8")
+    (crate_dir / "README.md").write_text("# demo", encoding="utf-8")
+    (crate_dir / "Cargo.toml").write_text(
+        "\n".join(
+            (
+                "[package]",
+                'name = "rstest-bdd-harness-gpui"',
+                'description = "demo"',
+                'readme = "README.md"',
+            )
+        ),
```

Ruff (0.15.5)

[error] Consider f-string instead of string join

Replace with f-string

(FLY002)